jenkins/validate-artifacts/validate-artifacts.jenkinsfile

```diff
@@ -65,6 +76,19 @@ pipeline {
             trim: true,
             defaultValue: "using-staging-artifact-only"
         )
+        string(
+            name: 'PROJECTS',
+            description: 'Enter opensearch or opensearch-dashboards',
+            trim: true,
+            defaultValue: "opensearch"
+        )
```

Comment:
Should this be a `CHOICE`?

Reply:
Not sure if that's required. It takes values like opensearch or opensearch opensearch-dashboards and validate accordingly.

Reply:
Does it take both or either? I believe if it want to validate both , there should be an option for that as well? 
I think right option here is `OPENSEARCH` OR `ALL` since we need both os and osd for osd to be validated? What do you think? OSD cannot be validated on its own right?

Reply:
both or either(only opensearch). Yes that case is covered here. The input could be `opensearch` or `opensearch opensearch-dashboards`. Both the scenarios are validated here @gaiksaya

Reply:
Sorry if that was not clear. When options are provided to the user I believe it should be `OPENSEARCH` OR `ALL` instead of `opensearch or opensearch-dashboards`